UploadOrganizationLogo/handler.ts

```diff
@@ -0,0 +1,132 @@
+import * as express from "express";
+
+import {
+  ClientIp,
+  ClientIpMiddleware
+} from "io-functions-commons/dist/src/utils/middlewares/client_ip_middleware";
+
+import { RequiredParamMiddleware } from "io-functions-commons/dist/src/utils/middlewares/required_param";
+
+import {
+  AzureApiAuthMiddleware,
+  IAzureApiAuthorization,
+  UserGroup
+} from "io-functions-commons/dist/src/utils/middlewares/azure_api_auth";
+import {
+  AzureUserAttributesMiddleware,
+  IAzureUserAttributes
+} from "io-functions-commons/dist/src/utils/middlewares/azure_user_attributes";
+import {
+  withRequestMiddlewares,
+  wrapRequestHandler
+} from "io-functions-commons/dist/src/utils/request_middleware";
+import {
+  IResponseErrorForbiddenNotAuthorized,
+  IResponseErrorInternal,
+  IResponseErrorNotFound,
+  IResponseErrorTooManyRequests,
+  IResponseSuccessJson,
+  ResponseSuccessJson
+} from "italia-ts-commons/lib/responses";
+import { OrganizationFiscalCode } from "italia-ts-commons/lib/strings";
+
+import {
+  checkSourceIpForHandler,
+  clientIPAndCidrTuple as ipTuple
+} from "io-functions-commons/dist/src/utils/source_ip_check";
+
+import { Context } from "@azure/functions";
+import { identity } from "fp-ts/lib/function";
+import { TaskEither } from "fp-ts/lib/TaskEither";
+import { ServiceModel } from "io-functions-commons/dist/src/models/service";
+import { ContextMiddleware } from "io-functions-commons/dist/src/utils/middlewares/context_middleware";
+import { RequiredBodyPayloadMiddleware } from "io-functions-commons/dist/src/utils/middlewares/required_body_payload";
+import { APIClient } from "../clients/admin";
+import { Logo } from "../generated/api-admin/Logo";
+import { withApiRequestWrapper } from "../utils/api";
+import { getLogger, ILogger } from "../utils/logging";
+import { ErrorResponses, IResponseErrorUnauthorized } from "../utils/responses";
+
+type ResponseTypes =
+  | IResponseSuccessJson<undefined>
+  | IResponseErrorUnauthorized
+  | IResponseErrorForbiddenNotAuthorized
+  | IResponseErrorNotFound
+  | IResponseErrorTooManyRequests
+  | IResponseErrorInternal;
+
+const logPrefix = "UploadOrganizationLogoHandler";
+
+/**
+ * Type of a UploadOrganizationLogoHandler handler.
+ *
+ * UploadOrganizationLogo expects an organization fiscal code and a logo as input
+ * and returns informations about upload outcome
+ */
+type IUploadOrganizationLogoHandler = (
+  context: Context,
+  auth: IAzureApiAuthorization,
+  clientIp: ClientIp,
+  attrs: IAzureUserAttributes,
+  organizationFiscalCode: OrganizationFiscalCode,
+  logoPayload: Logo
+) => Promise<ResponseTypes>;
+
+const uploadOrganizationLogoTask = (
+  logger: ILogger,
+  apiClient: APIClient,
+  organizationFiscalCode: OrganizationFiscalCode,
+  logo: Logo
+): TaskEither<ErrorResponses, IResponseSuccessJson<undefined>> =>
```

Comment:
SInce we are returning `undefined`, is that a hint that we might return something else (like 202 o0r 204)?